scripts.js

```diff
@@ -178,7 +173,9 @@ var app = {
 		var repCardList = document.getElementById('repList');
 
 		// Remove hidden class
-		repContainer && (repContainer.classList = '');
+		if (repContainer) {
+			repContainer.classList = '';
+		}
```

Comment:
Don't think this will fix it. Couldn't recreate. Figured I would standardize it to cut out the possibility of Safari not being able to handle the different syntax.